package.json

```diff
@@ -56,6 +56,8 @@
     "eslint-plugin-n": "^16.3.1",
     "eslint-plugin-no-only-tests": "^3.1.0",
     "eslint-plugin-perfectionist": "^2.4.0",
+    "eslint-plugin-react": "^7.33.2",
+    "eslint-plugin-react-hooks": "^4.6.0",
```

Comment:
I'd like to keep them as optional peer deps, and ask ppl to explicitly install them when they want to use react. This way we don't introduce those deps to every user of this config.

Reply:
I moved them to optional peer deps. Should I leave them in devDeps for dev?